bgpd/bgp_route.c

```diff
@@ -7396,11 +7396,13 @@ void route_vty_out_tmp(struct vty *vty, struct prefix *p, struct attr *attr,
 		json_object_boolean_true_add(json_status, ">");
 		json_object_object_add(json_net, "appliedStatusSymbols",
 				       json_status);
-		char buf_cut[BUFSIZ];
-		json_object_object_add(
-			json_ar,
-			inet_ntop(p->family, &p->u.prefix, buf_cut, BUFSIZ),
-			json_net);
+		char buf1[BUFSIZ];
```

Comment:
Won't this break existing people using this json?  Shouldn't we just add a new key?